### ISSUE TITLE
Create private account by default

### ISSUE THREAD

**Issue by blopp3** [nuxsmin/sysPass#793]
Hello there , 

I'm trying to figure out how to enable private accounts by default , we want to use it in our organisation but we at this point every new accounts are public , is there a way to configure a group or a profile without specify every time "private access" ? 

Thanks and great job !

**Comment by CleemL** [comment]
Hi, 

You have to create a groupe, like 'private' where only SA is in or anything else, and when you create an account, you have to select 'private' in the "principal group" list; and you select all user or group who have access on the permissions.

**Comment by blopp3** [comment]
Thanks for your answer :)

I don't think you get what I meant , imagine , all users are all in one group called "users" for example. One of this user creates an account and forgot to check private access , every one in the company will be able to see it,  right ? 
My point is, I'd like to know if there's a way to set account creation private by default 
![syspass](https://user-images.githubusercontent.com/33963034/33610061-9baa41ea-d9ca-11e7-9529-910c0a3a7cf0.PNG)
Like this but without being obliged to select private

**Comment by nuxsmin** [comment]
Hello @blopp3 , that is am useful feature that has been requested a time ago. It could be implemented in next version.

I'm glad to know you found sysPass useful ;)

Thanks for the feedback!

**Comment by dasunsrule32** [comment]
Is there a flag to set in a configuration file to enable 'Private' by default? I the following:

``account_isPrivate` bit(1) DEFAULT b'0',`

Would it help to set that to 1 in the database? Thanks :)

**Comment by nuxsmin** [comment]
@dasunsrule32 that's right, but it will lead to undesired behaviour, like customers assignment, since a customer assigned to a private account will be only visible by its owner, so you will need to create as many customers as users are registered.

**Comment by dasunsrule32** [comment]
@nuxsmin Hrm, ok. I'll just train them to mark it private by default and then share what they need to share. Are there any plans for a Chrome/Firefox plugin where we could set the syspass server and pull down credentials and autofill from the database? :)

**Comment by nuxsmin** [comment]
@dasunsrule32 sure, BTW global customers will be implemented in next release ;)

I'm currently working on sysPass 3.0 but an extension is the next plan then, hopefully, I'll give up lastpass :)

**Comment by dasunsrule32** [comment]
@nuxsmin Sweet :) 👍

**Comment by nuxsmin** [comment]
Hello, this feature has been implemented in v3. Thanks for the feedback!

Regards

**Comment by thalassa-web** [comment]
So it's implemented … But how do I enable this option ?
Whene I create an account, it's not private by default … So I looked into admin preferences, accounts preferences … And I can't find anything !
Did I miss something ?

BTW, thanks for the product, it was easy to deploy and use on my own VPS !

**Comment by thalassa-web** [comment]
OK ! Found !
I post what I've done for anyone else ^^
Sorry for the terms I'll use, I use the app in french so I tried to translate ^^ …

1. Connect with admin account
2. Go to «Elements and personalization» (the circle with 3 dots)
3. Go to «Predefined values» (the last tab)
4. Click on the vertical 3 dots (not the top of the page one, but just above the table)
5. Click on «preset private accounts»
6. Select a specific user / group or profile in the «general» tab
7. In the second tab, enable the option «private account»
8. Save

Now, all users in the specified group / profile will have the «private account» option activated by default !
